app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -2442,15 +2444,15 @@ public static String extraStatusLine() {
             if (prefs.getBoolean("status_line_carbs", false)) {
```

Comment:
Just UI improvements to make it consistent with other status line elements where there is no space after the label.